UnityProject/Assets/Scripts/UI/Net/Elements/Dynamic/NetUIDynamicList.cs

```diff
@@ -115,10 +116,15 @@ public virtual void Clear()
 	/// </summary>
 	private void RearrangeListItems()
 	{
-		if (!MasterTab.IsServer) return;
-		NetworkTabManager.Instance.Rescan(MasterTab.NetTabDescriptor);
-		RefreshPositions();
-		UpdatePeepers();
+		if (MasterTab.IsServer)
+		{
+			NetworkTabManager.Instance.Rescan(MasterTab.NetTabDescriptor);
+			RefreshPositions();
+			UpdatePeepers();
+		}
+
+		// rebuild layout to fix bug with moved UI elements position
+		LayoutRebuilder.MarkLayoutForRebuild(transform as RectTransform);
```

Comment:
oh cool, does that fix the need to open tabs twice?

Reply:
If you are about bug when vendors UI moves to the wrong corner after first opening - it is.